src/reducers/index.js

```diff
@@ -1,15 +1,15 @@
-import {uuidv1 as uuid} from 'uuid/v1'
+import {v1 as uuid} from 'uuid'
 
-export const initialState = {
+export const initialState = [{
     item: 'Learn about reducers',
     completed: false,
```

Comment:
I would also suggest adding a error to state like 
`const intialState = {
    todo: [],
    loading: false,
completed: false,
    error: ''
}`

You can dispatch errors from your actions as well.